source/material_model/multicomponent.cc

```diff
@@ -316,6 +272,20 @@ namespace aspect
     }
 
 
+    template <int dim>
+    std::vector<double>
+    Multicomponent<dim>::get_vector_double (const std::string &parameter, const unsigned int n_fields, ParameterHandler &prm)
```

Comment:
This function doesn't actually need to be a member function. Can you just move it to an anonymous namespace instead?